combineRooms/combineRooms.py

```diff
@@ -0,0 +1,320 @@
+            done = await self._start_combines(ctx)
+        elif action in ["stop", "teardown", "end"]:
+            done = await self._stop_combines(ctx)
+        else:
+            combines_ongoing = await self._combines_category(ctx.guild)
+            if combines_ongoing:
+                done = await self._stop_combines(ctx)
+            else:
+                done = await self._start_combines(ctx)
+        
+        if done:
+            await ctx.send("Done")
+        return
+    
+    @commands.command(aliases=["sppr"])
+    @commands.guild_only()
+    @checks.admin_or_permissions(manage_guild=True)
+    async def set_players_per_room(self, ctx, size: int):
+        if size < 2:
+            await ctx.send("No.")
+            return False  
+        combines_cat = await self._save_players_per_room(ctx.guild, size)
+        # DISABLED: room size in name
+        if combines_cat and False:
+            for vc in combines_cat.voice_channels:
+                pass # await self._adjust_room_(guild, vc)
+        await ctx.send("Done")
+        return True
+    
+    @commands.command(aliases=["ppr"])
+    @commands.guild_only()
+    async def get_players_per_room(self, ctx):
+        size = await self._players_per_room(ctx.guild)
+        await ctx.send("Combines should have no more than {0} active players in them.".format(size))
+
+    @commands.command(aliases=["setroomcap", "src"])
+    @commands.guild_only()
+    @checks.admin_or_permissions(manage_guild=True)
+    async def set_room_capacity(self, ctx, size: int):
+        if size < 2:
+            await ctx.send("No.")
+            return False  
+        combines_cat = await self._save_room_capacity(ctx.guild, size)
+        if self._combines_category(ctx.guild):
+            await ctx.send("Done: Changes will not be applied to rooms that are already up.")
+        else:
+            await ctx.send("Done")
+        return True
+
+    @commands.Cog.listener("on_voice_state_update")
+    async def on_voice_state_update(self, member, before, after):
+        combines_ongoing = await self._combines_category(member.guild)
+
+        # ignore when combines are not ongoing, or when voice activity is within the same room
+        if not combines_ongoing or before.channel == after.channel:
+            return
+        
+        # Room joined:
+        await self._member_joins_voice(member, after.channel)
+        
+        # Room left:
+        await self._member_leaves_voice(member, before.channel) # TODO: consider disconnected case #@me what does that even mean? this structure should cover everything
+
+    async def _start_combines(self, ctx):
+        # Creates combines category and rooms for each tier
+        combines_category = await self._add_combines_category(ctx, self._combines_category_name)
+        await self._save_combine_category(ctx.guild, combines_category)
+
+        if combines_category:
+            await self._add_combines_info_channel(ctx.guild, "Combines Details")
```

Comment:
A little inconsistency here with the hard-coded name "Combines Details". On line 97 you use a class variable as the name rather than hard-coding it in-line. I'd do one or the other on both lines to be consistent. I don't particularly think either needs to be a class variable, but it might be the proper way. Up to you.

Reply:
Changed it to be hard-coded.

We can make changes in the future if we deem it necessary. I believe I was thinking along the lines of having more customization options, but for now it'll be nice to avoid the clutter.